### DISCUSSION TITLE
Audio Gridder failed to bind to the server port (55056)

### DISCUSSION THREAD

**Discussion by mcaudio** [apohl79/audiogridder#707] [Q&A] [answered]
I go in to task manager and shut down one instance and nothing happens. Shut down both instances and it keeps re-opening two new instances. Eventually I can shut them both down but unable to open AG at all after that. Any thoughts? I did try both stable and Beta versions.

**Comment by apohl79** [comment]
Does it happen when you restart the server? Or does it always happen? In the later case another process is using the port. You can use something like sysinternals tcpview to figure out what process that is. 

You can make AG use a different port, by assigning a different ID to your server in the settings. The port AG uses is 55056 + ID, where the default ID is 0. If you assign ID 1, it will bind to 55057.

**Comment by mcaudio** [comment]
First, older computer that I never liked. Bought a new server computer today. Should eliminate a lot of issues. I installed the server software in my daw computer and it seems it will work. I di, however, have serious problems trying to get it to recognize Waves version 12 plugins. Never got it done. Reinstalling tomorrow in the new server. I'll let you know how it goes. Your response is very much appreciated! This is a great idea and fantastic for those of us longing f6it. I'll be glad to toss some cash your way if we can make this work. Thanks for all you've already done! I'll be in touch.

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by mcaudio** [comment]
I installed the new computer tonight. A server. Everything so far has worked flawlessly! In the next couple days I'll move my Waves plugins to the server. The Abbey Road plates and chambers have really bogged down my daw computer in the past to the point where I can barely use them. Anxious to see how it works. They are CPU hogs!
What you have designed is amazing! I hope you make amazing amounts of money with it. I will be contributing all I can afford which is not a ton but I hope it helps. And I'll try to contribute every time I update.
Thank you for the product and for following up so professionally!

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Reply by apohl79** [reply]
Thanks man! The abbey road collection works great for me.

**Comment by mcaudio** [comment]
Good to hear! I'll certainly let you know.On Sep 5, 2021 3:12 AM, Andreas Pohl ***@***.***> wrote:
Thanks man! The abbey road collection works great for me.

**Comment by mcaudio** [comment]
Just can't get the server to scan and find the Waves plugins. Looking for information but striking out. What am I doing wrong? Totally legitimate installs. AG just not finding them.

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by mcaudio** [comment]
Got it! Oh yeah!

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________

**Comment by Fittkenny** [comment]
Can you perhaps stop the Outlook for Android posts please?

**Comment by mcaudio** [comment]
Not sure where they are coming from. Seriously trying. Sorry for the annoyance. I think my server is doing it.

________________________________

**Comment by mcaudio** [comment, marked as answer]
Had to install new computer. Where do I copy waveshells to so AG recognizes them?

**Comment by Henri75** [comment]
This problem stills happens on Windows only.
Never got it on first launch, sometimes when I restart it, and often after a server crashes when the server restart itself !
A bit annoying as without sandboxes the server crash quite often.

**Comment by mcaudio** [comment]
Don't know if this will help, but I reinstalled and made sure User Account Control was wide open. After installing I brought it back down to zero. Good luck.

Michael
________________________________
